doc/luasnip.txt

```diff
@@ -634,6 +634,8 @@ API-REFERENCE                                              *luasnip-api-referenc
 *   `expand()`: expands the snippet at(before) the cursor.
 *   `expand_or_jumpable()`: returns `expandable() or jumpable(1)` (exists only
     because commonly, one key is used to both jump forward and expand).
+*   `expand_or_locally_jumpable()`: same as `expand_or_jumpable()` except jumpable
+    is ignored if the cursor is not inside a snippet. 
```

Comment:
`doc/luasnip.txt` is auto-generated from `DOC.md`, better add stuff there, this will be overwritten otherwise :P

Reply:
Also, I'd slightly prefer `the current snippet` to `a snippet` as with history enabled, there are many (active) snippets.

Reply:
> Also, I'd slightly prefer `the current snippet` to `a snippet` as with history enabled, there are many (active) snippets.

Good point!